src/translations/en_translations.json

```diff
@@ -30,7 +30,7 @@
   "standUpColoradoCta": "Call the Stand Up Colorado Helpline at",
   "guidedSearch": "Guided search",
   "next": "Next question",
-  "previousQuestion": "Back to the previous question",
+  "previousQuestion": "Back",
```

Comment:
I think "Back" gets my vote over ("Previous question") since the "Next question" above provides context